javascript/javascript1/week4/homework6/scripts.js

```diff
@@ -0,0 +1,133 @@
+let todos = []; // initializing an empty list of todo
+let loggedName = []; //initializing an empty list of names
+
+//function to capture the name from the command
+function getName(command){
+    const name = command.split(' ')[4]; // note to self, split is a string method, it takes a separtor argument, in this case it's a space, and returns an array populated with the separeted elements as strings, I am accessing the 5th element directly
+    return name
+}
+
+//function to capture a task, either to add or remove
+function getTask(command) {
+    if(command.startsWith('Add') || command.startsWith('add')){
+    const endIndex = command.indexOf(' to my todo') // I plan on using slice later, so I'm getting the index at which the slice should stop
+    const task = command.slice(4, endIndex) // get the part of the string between "Add" and "to my todo"
+    return task}
+    //do the same thing except adjust for the words "from" and "Remove"
+    else if (command.startsWith('Remove') || command.startsWith('remove')){
+    const endIndex = command.indexOf(' from my todo') 
+    const task = command.slice(7, endIndex) 
+    return task
+    }
+}
+
+//function to remove a task from the todos
+function removeTask(task) {
+    const taskIndex = todos.findIndex(task);
+    todos.slice(taskIndex, 1)
+    return console.log(`Removed ${task} from your todo`)
+}
+
+// function print date in human redable format
+function getDate(){
```

Comment:
There's a neat (newish) method in Javascript called `.toLocaleString()` that will format a date based on some options you provide to it. It could be easier than doing the formatting this way.

For example:
```
let date = new Date();
let dateFormatOptions = {
    month: "long",
    year: "numeric"
};

return date.toLocaleString("en", dateFormatOptions);
```
I'll let you figure out the rest 😉

Reply:
Many thanks 🙏